pkg/types/validation/installconfig.go

```diff
@@ -89,8 +89,7 @@ func ValidateInstallConfig(c *types.InstallConfig) field.ErrorList {
 		nameErr = validate.ClusterName1035(c.ObjectMeta.Name)
 	}
 	if c.Platform.Ovirt != nil {
-		// FIX-ME: As soon bz#1915122 get resolved remove the limitation of 14 chars for the clustername
-		nameErr = validate.ClusterNameMaxLength(c.ObjectMeta.Name, 14)
+		nameErr = validate.ClusterName(c.ObjectMeta.Name)
 	}
```

Comment:
At line 87 we already check for `validate.ClusterName`, so this `if` can actually be removed.

Reply:
You are right, removing it. Thanks!